app/src/main/java/com/github/se/eventradar/model/event/Event.kt

```diff
@@ -11,13 +11,19 @@ enum class EventCategory {
   COMMUNITY
 }
 
+data class Ticket(val name: String, val price: Double, val quantity: Int)
+// new event struct final
 data class Event(
-    val id: String,
-    val hostUserId: String,
-    val name: String,
+    val eventName: String,
+    val eventPhoto: String,
+    val start: LocalDateTime,
+    val end: LocalDateTime,
+    val location: Location,
     val description: String,
-    val date: LocalDate,
-    val time: LocalTime,
-    val category: EventCategory
-    // TODO: Add 'location' attribute once the Map feature is implemented
+    val ticket: Ticket,
+    val contact: String,
```

Comment:
suggestion: I think this should be a reference to the uid of the Host, not just a contact email string.